### PR TITLE
chore: add ingress-nginx to docs

### LINKED ISSUE #45: ingress-nginx support
Hi 👋🏼  It would be pretty neat to have your project support [ingress-nginx](https://kubernetes.github.io/ingress-nginx/), currently I've been using [181192/custom-error-pages](https://github.com/181192/custom-error-pages) which has some of your themes (there's credit to you in the README). However it would be awesome if your project was a solution for both traefik and ingress-nginx. Thanks!

Edit: This is a very basic implentation of the `default-backend` as provided by the ingress-nginx project.
https://github.com/kubernetes/ingress-nginx/tree/main/images/custom-error-pages/rootfs

### PR DESCRIPTION
**Pull request by onedr0p**
## Description

Added docs on ingress-nginx

Fixes #45

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

**Comment by tarampampam** [comment]
I want to ask you one more thing - could you please fix my mistakes in the readme file? I suppose there are a lot of them :/

**Comment by onedr0p** [comment]
Not sure what you mean, the readme looks fine as is 😄

File: README.md [hunk 1]
```diff
@@ -407,6 +407,30 @@ After executing `docker-compose up` you can:
...
+```yaml
+controller:
+  config:
+    custom-http-errors: >-
+      401,403,404,500,501,502,503
+defaultBackend:
+  enabled: true
+  image:
+    repository: ghcr.io/tarampampam/error-pages
+    tag: latest
```
Comment (tarampampam) [inline README.md:427]:
```suggestion
    tag: latest # Using the latest tag is highly discouraged. Please, use tags in X.Y.Z format
```

File: README.md [hunk 2]
```diff
@@ -407,6 +407,30 @@ After executing `docker-compose up` you can:
...
+  # optional: change the default theme
+  extraEnvs:
+  - name: TEMPLATE_NAME
+    value: l7-dark
```
Comment (tarampampam) [inline README.md:431]:
```suggestion
    value: l7-dark
  - name: SHOW_DETAILS # Optional: enables the output of additional information on error pages
    value: 'true'
```

File: README.md [hunk 3]
```diff
@@ -407,6 +407,30 @@ After executing `docker-compose up` you can:
...
+    custom-http-errors: >-
+      401,403,404,500,501,502,503
+defaultBackend:
+  enabled: true
+  image:
+    repository: ghcr.io/tarampampam/error-pages
+    tag: latest
+  # optional: change the default theme
+  extraEnvs:
+  - name: TEMPLATE_NAME
```
Comment (tarampampam) [inline README.md:430]:
```suggestion
  extraEnvs:
  - name: TEMPLATE_NAME # Optional: change the default theme
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Review by tarampampam — APPROVED** [review]

**Pull request merged**